modular_skyrat/modules/emotes/code/emotes.dm

```diff
@@ -104,15 +104,25 @@
 	message = "lets out an awoo!"
 	emote_type = EMOTE_AUDIBLE
 	vary = TRUE
-	sound = 'modular_skyrat/modules/emotes/sound/voice/awoo.ogg'
+
+/datum/emote/living/awoo/run_emote(mob/user, params, type_override, intentional)
+	. = ..()
+	for(var/i in view(1, user))
```

Comment:
Good PR but I think this should be viewers. View can be any atom, while viewers is any mob.

Reply:
Good point, should update this soon.